src/js/browser-plugin.js

```diff
@@ -229,43 +327,53 @@ const createWhyButton = (publicationName, addPaddingTop = false) => {
           resolve();
         })
       }
-    }).then(function (result) {
-
-      return new Promise((resolve) => {
-        var returned = result[Object.keys(result)[0]];
-        returned = returned.toString();
-        var array = returned.split(',');
-        var url = array[0], comment = array[1];
-
-        if (url.localeCompare('') === 0) {
-          alert('You need to insert an URL!');
-        }
-        else if (comment.localeCompare('') === 0) {
-          alert('You need to insert a comment!');
-        }
-        else {
-          // url comment resultDropdown
-          // make api call here
-          var evaluation = { 'label': resultDropdown, 'url': url, 'comment': comment};
+    }).then(function(result) {
+      if (result.value) {
+        return new Promise((resolve) => {
+          var returned = result[Object.keys(result)[0]];
+          returned = returned.toString();
+          var array = returned.split(',');
+          var url = array[0], comment = array[1];
 
-          client.postTwitterEvaluate(tweetData.id, evaluation)
-          .then(res => {
-            
-            console.log('Query ID = ' + JSON.stringify(res));
+          if (url.localeCompare('') === 0) {
+            alert('You need to insert an URL!');
+          }
+          else if (comment.localeCompare('') === 0) {
+            alert('You need to insert a comment!');
+          }
+          else {
             
-          })
-          .catch(err => console.log(err));
+            Swal2.fire('Sent!', 'Your feedback has been sent.', 'success')
 
-          resolve();
-        }
-      })
+            // url comment resultDropdown
+            var evaluation = { 
+              'evaluation': [ { 'label': resultDropdown, 'url': url, 'comment': comment}]
+            }
+            
+            client.postTwitterEvaluate(tweetData.id, evaluation)
+            .then(res => {
+              
+              console.log('Query ID = ' + JSON.stringify(res));
+              
+            })
+            .catch(err => console.log(err));
+
+            resolve();
+          }  
+        })
+      } else if (result.dismiss == 'cancel') {
```

Comment:
Use `===` instead of `==` when comparing in JavaScript.

Reply:
Changed